Update to latest Lobsters

ellie opened the pull request:
Both the Lobsters and Ruby versions here were running a little behind, so I updated them 🦞

ellie committed:
Update lobsters to latest HEAD

ellie committed:
Update Ruby to 2.7

doomspork commented:
Howdy @ellie :wave: Thanks for taking the time to do this and open a PR! I'll let our masterful @jamesbrink take a peek to make sure there aren't any hiccups and we can get this merged in :rocket:

ellie commented on Dockerfile:
```diff
@@ -49,7 +49,6 @@ RUN set -xe; \
     su lobsters -c "gem update"; \
     su lobsters -c "gem install rake -v 12.3.2"; \
     su lobsters -c "bundle install --no-cache"; \
-    su lobsters -c "bundle add puma --version '~> 3.12.1'"; \
```
Lobsters already specifies Puma in its Gemfile, hence I've removed it here :)

jamesbrink replied:
awesome. I had to go digging into the puma thing, I believe it was broken upstream for a while and thats how it landed in there.. good catch. it's def no longer needed 

https://github.com/utensils/docker-lobsters/commit/9b011778e4a2d808c6ee073c3cffa08e2cecba5f#diff-3254677a7917c6c01f55212f86c57fbfL58

ellie commented:
Hey @doomspork! Thanks for the speedy reply! Seems to be running OK, but let me know if anything needs changing 😄

jamesbrink committed:
Add small fixes related to ruby 2.7 update

Updated the README.md to reference new ruby version.
Updated the Makefilew to use ruby:2.7 as default base image.
Updated the Dockerfile to use 2.7 gems path and bumped rake to 13.0.1

jamesbrink commented:
thanks @ellie ! i ran into one small issue with rake mostly, I opened a PR against your branch. Could you verify that works for you and I did not muck something up here

ellie committed:
Merge pull request #1 from utensils/small_fixes

Add small fixes related to ruby 2.7 update

ellie committed:
Update PATH for new Ruby version

jamesbrink reviewed (approved):
All looks good @ellie thank you very much! 🎉 💪

ellie commented:
Awesome, thanks @jamesbrink! Appreciate the speedy replies from both of you 🎉🚀
